casper/src/test/scala/coop/rchain/casper/MultiParentCasperMergeSpec.scala

```diff
@@ -146,70 +146,69 @@ class MultiParentCasperMergeSpec extends FlatSpec with Matchers with Inspectors
+    "!X 4!!"    -> HAD_ITS_MATCH              -> merges(S0, F_, R0),
+    "!X (4!!)"  -> PERSISTENT_COULD_NOT_MATCH -> merges(S0, F_ | R0, Nil),
+    "!X !!X"    -> SAME_POLARITY_MERGE        -> merges(S0, R0, Nil),
+    "!X !!4"    -> SAME_POLARITY_MERGE        -> merges(S0, R1, F1),
     "!X (!!4)"  -> PERSISTENT_COULD_NOT_MATCH -> coveredBy("!X (4!!)"),
     "!X CX"     -> INCOMING_COULD_MATCH       -> conflicts(S0, C_, Nil),
     "!X C!"     -> INCOMING_COULD_MATCH       -> conflicts(S0, C_, S0),
     "!X (C!)"   -> PERSISTENT_COULD_MATCH     -> conflicts(S0, C_ | S0, Nil),
     "!4 !4"     -> COULD_MATCH_SAME_TRUE      -> conflicts(S0, S1, F_),
-    "!4 !4"     -> COULD_MATCH_SAME_FALSE     -> conflictsForNow(S0, S1, F0 | F1),
+    "!4 !4"     -> COULD_MATCH_SAME_FALSE     -> merges(S0, S1, F0 | F1),
     "!4 (!4)"   -> VOLATILE_EVENT             -> merges(S0, S1 | F_, F0),
     "(!4) (!4)" -> VOLATILE_EVENT             -> merges(S0 | F_, S0 | F_, Nil),
-    "!4 !C"     -> COULD_MATCH_SAME_FALSE     -> conflictsForNow(S0, S1, F0 | C1),
-    "!4 4X"     -> HAD_ITS_MATCH              -> conflictsForNow(S0, F_, F_),
-    "!4 4!"     -> HAD_ITS_MATCH              -> conflictsForNow(S0, F_, F0 | S1),
+    "!4 !C"     -> COULD_MATCH_SAME_FALSE     -> merges(S0, S1, F0 | C1),
+    "!4 4X"     -> HAD_ITS_MATCH              -> merges(S0, F_, F_),
+    "!4 4!"     -> HAD_ITS_MATCH              -> merges(S0, F_, F0 | S1),
     "!4 (4!)"   -> VOLATILE_EVENT             -> merges(S0, S1 | F_, F0),
-    "!4 4!!"    -> HAD_ITS_MATCH              -> conflictsForNow(S0, F_, F0 | R1),
-    "!4 !!X"    -> SAME_POLARITY_MERGE        -> conflictsForNow(S0, R1, F0),
-    "!4 !!4"    -> SAME_POLARITY_MERGE        -> conflictsForNow(S0, R1, F_ | F1),
-    "!4 CX"     -> HAD_ITS_MATCH              -> conflictsForNow(S0, C_, F_),
-    "!4 C!"     -> HAD_ITS_MATCH              -> conflictsForNow(S0, C_, F0 | S1),
-    "!4 (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> conflicts(S0, C_ | S0, F0),
-    "!4 (!C)"   -> PERSISTENT_COULD_NOT_MATCH -> conflicts(S0, S0 | C_, F0),
-    "!C !C"     -> COULD_MATCH_SAME_TRUE      -> conflictsForNow(S0, S0, C_),
-    "!C (!C)"   -> PERSISTENT_COULD_NOT_MATCH -> conflicts(S0, C_ | S1, C0),
-    "!C 4X"     -> HAD_ITS_MATCH              -> conflictsForNow(S0, F_, C_),
-    "!C 4!"     -> HAD_ITS_MATCH              -> conflictsForNow(S0, F_, C0 | S1),
+    "!4 4!!"    -> HAD_ITS_MATCH              -> merges(S0, F_, F0 | R1),
+    "!4 !!X"    -> SAME_POLARITY_MERGE        -> merges(S0, R1, F0),
+    "!4 !!4"    -> SAME_POLARITY_MERGE        -> conflicts(S0, R1, F_ | F1),
+    "!4 CX"     -> HAD_ITS_MATCH              -> merges(S0, C_, F_),
+    "!4 C!"     -> HAD_ITS_MATCH              -> merges(S0, C_, F0 | S1),
+    "!4 (C!)"   -> PERSISTENT_COULD_NOT_MATCH -> merges(S0, C_ | S1, F0),
+    "!4 (!C)"   -> PERSISTENT_COULD_NOT_MATCH -> merges(S0, S1 | C_, F0),
+    "!C !C"     -> COULD_MATCH_SAME_TRUE      -> merges(S0, S0, C_),
```

Comment:
this should be `COULD_MATCH_SAME_FALSE`. The FALSE refers to whether it conflicts or not. It used to be `same` vs `diff`. Maybe the cases should be named

COULD_MATCH_SAME_CONFLICT and COULD_MATCH_SAME_MERGE

Reply:
Good catch! Renaming sounds good

Reply:
Done